plugins/performance-lab/includes/server-timing/class-perflab-server-timing.php

```diff
@@ -67,7 +67,7 @@ public function register_metric( string $metric_slug, array $args ): void {
 			return;
 		}
 
-		if ( did_action( 'perflab_server_timing_send_header' ) && ! doing_action( 'perflab_server_timing_send_header' ) ) {
+		if ( 0 !== did_action( 'perflab_server_timing_send_header' ) && ! doing_action( 'perflab_server_timing_send_header' ) ) {
```

Comment:
This is such a common pattern in WP that I wouldn't even think of needing to specify, but makes sense.